app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -185,6 +189,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * get format of translation
+     * @return
+     */
+    public TranslationFormat getFormat() {
+        return mTranslationFormat;
+    }
+
+    private TranslationFormat readTranslationFormat() {
+        String formatStr = manifest.getString("format");
+        TranslationFormat format = TranslationFormat.get(formatStr);
+        if(null == format) {
```

Comment:
If the format is null then we need to check for the translation type first. Anything not a `text` translation type has a format of  `markdown`. If the type is `text` then if the project id is `obs` the format is `markdown`, otherwise the format is `usfm`.

We cannot completely depend on the resource type here because a `REGULAR` type can be either obs or a non-gateway language translation of a Bible project. 

> NOTE: In the android app we don't directly support translating to a `ulb` or `udb` resource type. However, the desktop app *does* support this. Therefore android provides just enough support so we can import a translation from the desktop app without breaking.